tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/store/debugger_selectors.ts

```diff
@@ -52,6 +51,7 @@ import {
 // https://github.com/bazelbuild/rules_nodejs/issues/1013
 /** @typehack */ import * as _typeHackSelector from '@ngrx/store/src/selector';
 /** @typehack */ import * as _typeHackStore from '@ngrx/store/store';
+import {connectableObservableDescriptor} from 'rxjs/internal/observable/ConnectableObservable';
```

Comment:
Unused. If you want to keep this, please put the typehack preamble.

Reply:
Ah - this is one of those annoying auto-imports due to VSCode + my typo. Removed. Thanks for catching it.